dvc_http/spec.py

```diff
@@ -0,0 +1,11 @@
+from fsspec.implementations.http import HTTPFileSystem
+
+
+class HTTPFileSystemWrapper(HTTPFileSystem):  # pylint: disable=abstract-method
```

Comment:
Let's keep the same name.
```suggestion
from fsspec.implementations import http


class HTTPFileSystem(http.HTTPFileSystem):  # pylint: disable=abstract-method
```